backend/Controllers/algorithmController.js

```diff
@@ -14,9 +14,9 @@ const redirect_uri = backend_url + "callback"; // Your redirect uri
 
 // instantiate spotifyApi object
 var spotifyApi = new SpotifyWebApi({
-    clientId: client_id,
-    clientSecret: client_secret,
-    redirectUri: redirect_uri,
+  clientId: client_id,
+  clientSecret: client_secret,
+  redirectUri: redirect_uri,
 });
```

Comment:
Is this the part where we connect to spotify API?

Reply:
yes that instantiates the spotifAPI object